### DISCUSSION TITLE
Load Worlds into one file and convert them from anvil format

### DISCUSSION THREAD

**Discussion by einfachBlu** [Minestom/Minestom#1533] [Help]
I know there is a world format called Slime. Unfortunately, this is not working anymore for Minestom. Is there any good alternative? It should be small so i can store the world file in my projects github repository. I also need to convert them from Anvil to one file. compressing it in a .zip or .rar file is not a solution for me so i am searching for something like slime format without creating it on my own.

My Worlds are not survival. They are Hubs, Lobbies, Minigame Maps like BedWars with lots of Void.

**Comment by Timofffee** [comment]
I think TNT will suit you
https://github.com/EmortalMC/TNT

**Comment by einfachBlu** [comment]
Also thought about it. The Repo is archived

**Reply by emortaldev** [reply]
Yes unfortunately, however it is still theoretically usable, just my anvil to TNT converter caused OOMs

**Reply by Timofffee** [reply]
@emortaldev ohh.. bad news. And as I understand it, there is no solution to be found?

**Reply by emortaldev** [reply]
Thought I would mention for future viewing: TNT has been rewritten in Java and no longer archived so it's not totally unsupported anymore lol
